### ISSUE TITLE
License

### ISSUE THREAD

**Issue by tony** [ffevotte/clang-tags#25]
Why a license different than clangs? Even the project your naming it after (clang) couldn't use the code.

GPLv3 is incompatible with basically every other license. One of the major reasons clang exists is not to deal with license problems anymore.

**Comment by ffevotte** [comment]
Yes, you're right. I did this out of habit, without even looking at the license LLVM and Clang themselves were using.

I will relicense to the same BSD-like license as LLVM.

Thanks for your comment!
